mlir/utils/widgets/tune_MLIR_kernels.sh

```diff
@@ -9,6 +9,7 @@ printUsage() {
     echo "  -c <\"config\">: Run MIOpenDriver on a single config"
     echo "                 Otherwise, on all configs in resnet50-miopen-configs"
     echo "  -u: turn off tuning (tuning is on be default)"
+    echo "  -b: don't run any tuning, just the compilation actions"
```

Comment:
[Question] What's the use case of this option?

Reply:
It's for when I want to build MIOpen but don't want to actually run any configs (ex. I'll be running MIOpenDriver under debugging)